feature/system/gnmi/set/tests/gnmi_set_test/gnmi_set_test.go

```diff
@@ -105,6 +135,9 @@ func TestGetSet(t *testing.T) {
 	p1 := dut.Port(t, "port1")
 	p2 := dut.Port(t, "port2")
 
+	gnmi.Update(t, dut, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Config(), &oc.NetworkInstance{
+		Name: ygot.String("DEFAULT"),
+	})
 	// Configuring basic interface and network instance as some devices only populate OC after configuration.
```

Comment:
Network instance type has to be configure to "DEFAULT", this will be configured along with line at 144 ? 
This is redundant. 

gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Type().Config(),
		oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)

Reply:
its a different call, one is 
NetworkInstance(deviations.DefaultNetworkInstance(dut)).Type().Config() and another is
NetworkInstance(deviations.DefaultNetworkInstance(dut)).Config()

Reply:
this is a different call, 
its not NI.Type config

Reply:
As per revision "2022-11-18" to network instance yang , network instance type is mandatory. 
If you need to configure instance name, type is mandatory to set. 

you can use something like this configure both name and type at the same time.

gnmi.Update(t, d, defNiPath.Config(), &oc.NetworkInstance{
		Name: ygot.String(deviations.DefaultNetworkInstance(d)),
		Type: oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE,
	})

Reply:
Agreed, I have clubbed both under single call rather than separate for type and name.